src/scopes.rs

```diff
@@ -35,12 +35,15 @@ pub enum ScopeKind {
   Program,
   Module,
   Function,
+  Method,
   Block,
   Loop,
   Class,
   Switch,
   With,
   Catch,
+  Getter,
+  Setter,
```

Comment:
Do we really need new scope types? Shouldn't `ScopeKind::Function` be enough?

Reply:
your right, **we** don't need them for the current rules, actually.
Done 👍 
stay minimal for now.


_I'm just curious about doing something in the future, Shouldn't we care about plugin development and analyzing all kinds of scopes? My opinion is that we should proceed according to the needs_

Reply:
> I'm just curious about doing something in the future, Shouldn't we care about plugin development and analyzing all kinds of scopes? 

I'm not sure - ESLint doesn't recognize `Getter/Setter` scopes so maybe it's critical

> My opinion is that we should proceed according to the needs

I agree